scipy/fft/tests/test_numpy.py

```diff
@@ -135,8 +135,33 @@ def test_ihfft(self):
             x_herm, fft.ihfft(fft.hfft(x_herm, norm="ortho"),
                                  norm="ortho"))
 
+    def test_hfft2(self):
```

Comment:
Not really an issue introduced in this PR because the same thing applies to the prexisting`test_rfftn`, etc, but it seems a bit odd that a class named `TestFFT1D` has n-dimensional test cases. I guess it could just be renamed to something like `TestFFT` or `TestFFTBasic`

Do you think it is worth adding parametrization over dtypes to these? I see that some other tests have this, but most do not. If the r2c/c2r code path for other dtypes is already being tested elsewhere it is probably not necessary

Reply:
>it seems a bit odd that a class named `TestFFT1D` has n-dimensional test cases.

Yes, for whatever reason this is how numpy's tests are organised. I agree that it's strange. 

>Do you think it is worth adding parametrization over dtypes to these?

I don't see much benefit since the same code paths are exercised by the tests for the `rfft` type functions.